Xamarin.Essentials/Connectivity/Connectivity.android.cs

```diff
@@ -18,7 +18,9 @@ static void StartListeners()
 
             conectivityReceiver = new ConnectivityBroadcastReceiver(OnConnectivityChanged);
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
This [constant](https://developer.android.com/reference/android/net/ConnectivityManager.html#CONNECTIVITY_ACTION) has been deprecated in favor of https://developer.android.com/reference/android/net/ConnectivityManager.html#registerNetworkCallback(android.net.NetworkRequest,%20android.app.PendingIntent) 

IMHO this should be replaced as this means that future Android versions will not support this anymore.